pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: tus_client
 description: A tus client for dart allowing resumable uploads using the tus protocol.
-version: 1.0.2
+version: 1.0.3

```

Comment:
This is a breaking API change i.e. old implementations can't just upgrade so 2.0.0 will be needed